qemu/tests/cfg/seabios.cfg

```diff
@@ -32,5 +32,5 @@
         - bin_file:
             type = seabios_bin
             machine_type_remove = "Supported none pc q35"
-            Host_RHEL.m9:
-                bin_file_skip = "bios.bin"
+            Host_RHEL.m6, Host_RHEL.m7, Host_RHEL.m8:
```

Comment:
bin_file_skip = bios.bin
Host_RHEL.m6, Host_RHEL.m7, Host_RHEL.m8:
    bin_file_skip = ""

No need to update py file